ethcore/src/ethereum/mod.rs

```diff
@@ -94,6 +94,11 @@ pub fn new_mix<'a, T: Into<SpecParams<'a>>>(params: T) -> Spec {
 	load(params.into(), include_bytes!("../../res/ethereum/mix.json"))
 }
 
+/// Create a new Callisto chain spec
+pub fn new_callisto<'a, T: Into<SpecParams<'a>>>(params: T) -> Spec {
```

Comment:
I might be out of the loop here but what network is Callisto?

Reply:
I just reordered the networks here, see a few lines below.